identity/src/main/java/com/stripe/android/identity/networking/models/VerificationReportRequirement.kt

```diff
@@ -0,0 +1,35 @@
+package com.stripe.android.identity.networking.models
+
+import kotlinx.serialization.SerialName
+
+internal enum class VerificationReportRequirement(val value: String) {
```

Comment:
nit - if not handled in any other layer, you can use this serializer to prevent issues if new values are added on backend: https://github.com/stripe/stripe-android/blob/570179015f473ee5d9254500df297a37f5899375/stripe-core/src/main/java/com/stripe/android/core/model/serializers/EnumIgnoreUnknownSerializer.kt#L16

Reply:
thanks, this class is actually not needed - removed